api/src/test/java/org/openmrs/validator/ObsValidatorTest.java

```diff
@@ -513,4 +517,23 @@ public void validate_shouldPassValidationIfValueTextIsLessThanTheMaximumLength()
 		Assert.assertFalse(errors.hasFieldErrors("obsDatetime"));
 		Assert.assertFalse(errors.hasFieldErrors("valueText"));
 	}
+
+	@Test
+	public void validate_shouldFailValidationForDifferentObsPersonAndWithEncounterPatient() {
```

Comment:
please rename to `validate_shouldFailValidationIfObsAndEncounterPersonAreNotEqual`